models/doctorDetailModel.js

```diff
@@ -0,0 +1,82 @@
+const mongoose = require("mongoose");
+
+const Joi = require("joi");
+
+const doctorSchema = new mongoose.Schema({
+  doctorName: {
+    type: String,
+    required: true,
+  },
+  qualification: {
+    type: [String],
+    required: true,
+  },
+  Gender: {
+    type: String,
+    required: true,
+  },
+  clinicName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: Object,
+    required: true,
+    ref: "User",
```

Comment:
Please use email id instead of email object if you want to reference data from other collection